src/model/communityMember.ts

```diff
@@ -0,0 +1,30 @@
+import * as z from 'zod';
+export const CommunityMemberSchema = z.object({
+  fullname: z.string().min(1, { message: 'Full name is required' }),
+  bio: z.string().min(1).max(30),
```

Comment:
maybe we may add the min & max info also in the tutorial

Reply:
sure, done here 54609a6bc6a2bb43b72237cab6a572b1c8a21e51